examples/command/portals/kafka/instaclustr/application_team/producer.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
Having the commands inline inside `docker-compose.yml` is causing authentication errors.